defaults/aws_config.yaml

```diff
@@ -37,7 +37,7 @@ ami_db_scylla_user: 'scyllaadm'
 ami_loader_user: 'centos'
 # centos7 is used for monitor
 ami_monitor_user: 'centos'
-aws_instance_profile_name: 'qa-scylla-manager-backup-instance-profile'
+aws_instance_profile_name_db: 'qa-scylla-manager-backup-instance-profile'
```

Comment:
shouldn't add also value for `aws_instance_profile_name_loader` profile?

Reply:
no, since by default we don't add to it.